### ISSUE TITLE
to_cookie_string mutates CookieHash values

### ISSUE THREAD

**Issue by sethwoodworth** [jnunemaker/httparty#459]
https://github.com/jnunemaker/httparty/blob/6f4d6ea4a2707a4f1466f45bf5c67556cdbed2b7/lib/httparty/cookie_hash.rb#L19

```ruby
[20] pry(main)> h
=> {:session=>"91e25e8b-6e32-418d-c72f-2d18adf041cd", :"Max-Age"=>"15552000", :cart=>"91e25e8b-6e32-418d-c72f-2d18adf041cd", :httponly=>nil, :Path=>"/", :secure=>nil}
[21] pry(main)> h.to_cookie_string
=> "session=91e25e8b-6e32-418d-c72f-2d18adf041cd; Max-Age=15552000; cart=91e25e8b-6e32-418d-c72f-2d18adf041cd"
[22] pry(main)> h
=> {:session=>"91e25e8b-6e32-418d-c72f-2d18adf041cd", :"Max-Age"=>"15552000", :cart=>"91e25e8b-6e32-418d-c72f-2d18adf041cd"}
```

I would not have expected `to_cookie_string` to be removing keys from a CookieHash.  I understand removing keys that are not in `CLIENT_COOKIES` in the output of the method, but not mutating the parent object.

**Comment by jnunemaker** [comment]
That is really odd. I'm open to a PR that fixes it. Thanks for filing.

**Comment by mikeastock** [comment]
I think this has been resolved by #460
